src/components/menus/content-contextual-menu.tsx

```diff
@@ -213,7 +219,7 @@ const ContentContextualMenu = (props: ContentContextualMenuProps) => {
                         activeElement.description,
                         activeElement.elementUuid,
                         selectedDirectory?.elementUuid,
-                        activeElement.specificMetadata.type
+                        activeElement.specificMetadata.type as unknown as string
```

Comment:
if we are sur that we have always the type we can do this ?
String(activeElement.specificMetadata.type)